scikeras/wrappers.py

```diff
@@ -1611,6 +1653,75 @@ class KerasRegressor(BaseWrapper):
         **BaseWrapper._tags,
     }
 
+    def __init__(
+        self,
+        model: Union[None, Callable[..., tf.keras.Model], tf.keras.Model] = None,
+        *,
+        build_fn: Union[
+            None, Callable[..., tf.keras.Model], tf.keras.Model
+        ] = None,  # for backwards compatibility
+        warm_start: bool = False,
+        random_state: Union[int, np.random.RandomState, None] = None,
+        optimizer: Union[
+            str, tf.keras.optimizers.Optimizer, Type[tf.keras.optimizers.Optimizer]
+        ] = "rmsprop",
```

Comment:
```suggestion
        ] = "adadelta",
```

Reply:
Why is this change necessary as part of this PR? This is just copied from `BaseWrapper.__init__`. If we change the default, we change the behavior.

Reply:
I'll submit another PR with type changes to `__init__`.

`"rmsprop"` is the default for [keras.model.compile][1], so I suppose it makes at least a little sense. But it's not at all popular.

[1]:https://keras.io/api/models/model_training_apis/#compile-method

Reply:
> I'll submit another PR with type changes to __init__.

Great, thank you! Please also feel free to just leave it there and I can get to it at some point, I do want those changes.

> it's not at all popular

I hear you. Indeed the only reason it's the default is because that is `Model.compile`'s default. A discussion to change this is welcome, but I think we'll need some more pros other than the fact that it's not that popular, and which one we should choose to replace it.

Reply:
I think the consistency with Keras trumps any concerns I have.